licenses-binary/LICENSE-gpl-2.0.txt

```diff
@@ -0,0 +1,641 @@
+Apache Pinot (incubating)
```

Comment:
This is not the license file. This looks more like a NOTICE file?

Reply:
Thanks for catching it!